src/skin/popup.html

```diff
@@ -129,6 +129,7 @@ <h2 id="title-name"><span class="i18n_name"></span></h2>
       <span class="i18n_popup_instructions_no_trackers" data-i18n_contents_placeholders="<a target='_blank' title='i18n_what_is_a_tracker' class='tooltip' href='https://privacybadger.org/#What-is-a-third-party-tracker'>"></span>
       <span id="no-third-parties" class="i18n_popup_blocked" style="display:none"></span>
     </span>
+    <span id="instructions_firstparty_protections" class="i18n_popup_info_firstparty_protections" style="display:none"></span>
```

Comment:
CSS names should use dash separators going forward: `instructions-firstparty-protections`.

Reply:
Should probably show this section above the "X trackers blocked" section, so as not to come between it and the slider list.